libcontainerd/process_unix.go

```diff
@@ -86,6 +87,9 @@ func (p *process) sendCloseStdin() error {
 		Pid:        p.friendlyName,
 		CloseStdin: true,
 	})
+	if err != nil && strings.Contains(err.Error(), "not found") {
```

Comment:
check for `container not found` or `process not found` is more accurate, will do